src/workflows/bidsCopyRawFolder.m

```diff
@@ -40,13 +42,17 @@ function bidsCopyRawFolder(opt, deleteZippedNii, modalitiesToCopy)
   end
   opt = loadAndCheckOptions(opt);
 
+  cleanCrash();
```

Comment:
just an idea. Maybe you did it already, to insert cleanCrash function in other spatialPreprocessing scripts? Sorry if I'm saying something you have already done it.